internal/ingress/controller/template/template.go

```diff
@@ -407,6 +408,11 @@ func buildLocation(input interface{}, enforceRegex bool) string {
 	if enforceRegex {
 		return fmt.Sprintf(`~* "^%s"`, path)
 	}
+
+	if location.PathType != nil && *location.PathType == networkingv1beta1.PathTypeExact {
```

Comment:
Can it ever be `nil` here given we have https://github.com/kubernetes/ingress-nginx/pull/5324/files#diff-ece58b3bebec02a6d3173c8d8099d298R667?